config/core/build-configs.yaml

```diff
@@ -1033,6 +1040,17 @@ build_configs:
           x86_64:
             base_defconfig: 'x86_64_defconfig'
 
+  rust-next:
```

Comment:
BTW the implicit convention for the build config names is to use the `tree_branch` format, so in this case it should be `rust-for-linux_rust-next`.  That's no big deal as these names aren't sent to the API but they may be used in some command line arguments.  Also this isn't really documented, it's just how they've been named until now to make it easier to manage the file.  I don't know if you want to go through the effort of making another PR to tweak that, if so it's better to do it now than later before external dependencies start appearing.